sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -496,6 +527,41 @@ static ExtractKeyPhrasesResultCollection getExpectedBatchKeyPhrases() {
         return new ExtractKeyPhrasesResultCollection(extractKeyPhraseResultList, DEFAULT_MODEL_VERSION, textDocumentBatchStatistics);
     }
 
+    static ExtractSummaryResultCollection getExpectedExtractSummaryResultCollection() {
+        final ExtractSummaryResultCollection expectResultCollection = new ExtractSummaryResultCollection(
+            asList(getExpectedExtractSummaryResult()), null, null);
+        return expectResultCollection;
+    }
+
+    static ExtractSummaryResult getExpectedExtractSummaryResult() {
+        final TextDocumentStatistics textDocumentStatistics = new TextDocumentStatistics(67, 1);
+        final ExtractSummaryResult extractSummaryResult = new ExtractSummaryResult("0", textDocumentStatistics, null);
+
+        final IterableStream<SummarySentence> summarySentences = IterableStream.of(asList(
```

Comment:
just FYI this is can cause errors every time the model gets updated.... u could leave as is and update your tests every time the model changes, or you could just very that the values returned are not empty and in between the ranges we expect. i.e. length is never 0

Reply:
Do you mean the example like:

 if today the text is "ABC", offset is 0, length is 3, but after service update the model, the text becames "A", offset no change, length becomes 1.
 
 Which we don't want to verify the property values, but only check, text is not null or empty, offset and length both greate than 0?

Reply:
yeah. exactly. at the end, if the service gets smarter that is great! but it will also break our tests. 
Whatever u feel comfortable with :)